Lib/test/libregrtest/worker.py

```diff
@@ -25,7 +25,7 @@ def create_worker_process(runtests: RunTests, output_fd: int,
     if python_cmd is not None:
         executable = python_cmd
     else:
-        executable = [sys.executable]
+        executable = (sys.executable,)
```

Comment:
Mypy complained about this line because on line 26, the `executable` variable has type `tuple[str, ...]`, but here, the same name is being assigned to an object of type `list[str]`. We can make mypy happy by just making it a `tuple` in both the branches.